### PR TITLE
task 3

### PR DESCRIPTION
**Pull request by aksanakandr**

### CODE REVIEW

File: task.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
+"""
```
Comment (antohaUa) [inline task.py:1]: Still present linter errors:
(venv) [antoha@dlhn test_project]$ ./bin/lint ./T-HOMETASKS/L03/l03_h01_AksanaKondratovich.py
./T-HOMETASKS/L03/l03_h01_AksanaKondratovich.py:1:1: D205 1 blank line required between summary line and description
./T-HOMETASKS/L03/l03_h01_AksanaKondratovich.py:1:1: D400 First line should end with a period
./T-HOMETASKS/L03/l03_h01_AksanaKondratovich.py:3:1: RST203 Definition list ends without a blank line; unexpected unindent.
./T-HOMETASKS/L03/l03_h01_AksanaKondratovich.py:21:1: I003 isort expected 1 blank line in imports, found 0
./T-HOMETASKS/L03/l03_h01_AksanaKondratovich.py:62:40: C812 missing trailing comma
./T-HOMETASKS/L03/l03_h01_AksanaKondratovich.py:68:11: E225 missing whitespace around operator
./T-HOMETASKS/L03/l03_h01_AksanaKondratovich.py:69:3: E225 missing whitespace around operator
./T-HOMETASKS/L03/l03_h01_AksanaKondratovich.py:70:3: E225 missing whitespace around operator
./T-HOMETASKS/L03/l03_h01_AksanaKondratovich.py:73:14: E225 missing whitespace around operator
./T-HOMETASKS/L03/l03_h01_AksanaKondratovich.py:75:14: E225 missing whitespace around operator
./T-HOMETASKS/L03/l03_h01_AksanaKondratovich.py:79:9: E225 missing whitespace around operator
./T-HOMETASKS/L03/l03_h01_AksanaKondratovich.py:82:21: W292 no newline at end of file

File: task.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    ('Fiona', 40, 150, 'Jones', 'female'),
+    ('George', 24, 180, 'Lewis', 'male'),
+    ('Hannah', 22, 200, 'Williams', 'female'),
+    ('Ivan', 29, 160, 'Brown', 'male'),
+    ('Julie', 55, 90, 'Clark', 'female'),
+    ('Kenny', 38, 140, 'Harris', 'male'),
+    ('Luna', 55, 170, 'Smith', 'female'),
+    ('Mike', 55, 55, 'Johnson', 'male')
+]
+"""people.sort(key=lambda x:(x[1],x[4]))
```
Comment (antohaUa) [inline task.py:64]: better to delete such lines or leave them as comments

File: task.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+   print(people)"""
+
+people.sort(key=itemgetter(1, 4))
+people_new=people[2:-2]
+ma=0
```
Comment (antohaUa) [inline task.py:69]: Not clear variable names . Plus better to do it in one line aka
female_count, male_count = 0, 0

File: task.py [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+    ('Julie', 55, 90, 'Clark', 'female'),
+    ('Kenny', 38, 140, 'Harris', 'male'),
+    ('Luna', 55, 170, 'Smith', 'female'),
+    ('Mike', 55, 55, 'Johnson', 'male')
+]
+"""people.sort(key=lambda x:(x[1],x[4]))
+   print(people)"""
+
+people.sort(key=itemgetter(1, 4))
+people_new=people[2:-2]
```
Comment (antohaUa) [inline task.py:68]: No print of new cut list
> two elements and last two elements. Print this new list

File: task.py [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+ma=0
+fe=0
+for people in people_new:
```
Comment (antohaUa) [inline task.py:71]: Overhead to use such cycles for count. Moreover we have no lesson about cycles yet )
As I said in task description you have to try flat the list and then use list.count('female')  - clear and simple!

File: task3_new.py
Comment (aksanakandr) [inline task3_new.py:1]: @antohaUa 
please review update :)

File: task3_new.py [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+    ('Fiona', 40, 150, 'Jones', 'female'),
+    ('George', 24, 180, 'Lewis', 'male'),
+    ('Hannah', 22, 200, 'Williams', 'female'),
+    ('Ivan', 29, 160, 'Brown', 'male'),
+    ('Julie', 55, 90, 'Clark', 'female'),
+    ('Kenny', 38, 140, 'Harris', 'male'),
+    ('Luna', 55, 170, 'Smith', 'female'),
+    ('Mike', 55, 55, 'Johnson', 'male'),
+]
+people.sort(key=itemgetter(1, 4))
```
Comment (antohaUa) [inline task3_new.py:67]: Here better to use sorted call cause new list will be created. Now with current solution initial data modified. That's not the best practice

**Review by antohaUa — APPROVED** [review]

**Pull request merged**